src/hazelcore/SkelExprParser.mly

```diff
@@ -46,6 +47,7 @@
 %left FMINUS
 %left FTIMES
 %left FDIVIDE
+%left PLUSPLUS
```

Comment:
let's follow OCaml: https://caml.inria.fr/pub/docs/manual-ocaml/expr.html#ss:precedence-and-associativity
`^` is right associative and binds less tightly than `::`

Reply:
actually can you check all of our precedences here to make sure they are in the same order as OCaml